common/lib/dependabot/pull_request_creator/message_builder.rb

```diff
@@ -127,6 +128,7 @@ def pr_message
         truncate_pr_message(msg)
       rescue StandardError => e
         Dependabot.logger.error("Error while generating PR message: #{e.message}")
+        Dependabot.logger.error(e.backtrace&.join("\n"))
```

Comment:
seems to be the same thing repeated thrice (maybe more times in the same file)? DRY opportunity?